src/js/index.js

```diff
@@ -248,16 +248,59 @@ export const hpe = deepFreeze({
     },
   },
   checkBox: {
-    color: 'selected-text',
-    gap: 'small',
-    check: {
-      radius: '2px',
+    hover: {
+      border: {
+        color: undefined,
+      },
+      background: {
+        color: 'background-contrast',
+      },
     },
     border: {
+      color: 'border',
       width: '1px',
     },
+    check: {
+      radius: '2px',
+      extend: ({ theme, checked, indeterminate }) => `
+      background-color: ${
+        checked || indeterminate
+          ? theme.global.colors['green!']
+          : theme.global.colors.background[theme.dark ? 'dark' : 'light']
+      };
+      ${(checked || indeterminate) && 'border: none;'}
+        `,
+    },
+    icon: {
+      extend: 'stroke-width: 2px;',
+    },
+    gap: 'small',
+    toggle: {
+      background: 'background',
+      color: 'background',
+      knob: {
+        extend: ({ theme }) => `
+           box-shadow: ${
+              theme.global.elevation[theme.dark ? 'dark' : 'light'].small
+            };
+           border: 1px solid ${
+             theme.global.colors.border[theme.dark ? 'dark' : 'light']
+           }
+        `,
+      },
+      extend: ({ checked, theme }) => `
+        ${checked && `background-color: ${theme.global.colors['green!']};`}
+      `,
+    },
+    extend: ({ theme }) => `
+      width: 100%;
+      padding: ${theme.global.edgeSize.small};
+    `,
   },
   formField: {
+    content: {
+      pad: undefined,
+    },
```

Comment:
@halocline, are we ok with this change affecting all forms and not only the checkbox? is this how we will recommend consuming the component without any pad? or is it applied elsewhere and I'm missing something.

Reply:
My understanding is that this `content.pad` affects CheckBox, RadioButtonGroup, and RangeInput. Each of these will need the small pad re-applied which will account for the spacing and the background color for the active/hover states. For CheckBox, this has been applied on Line 297. RadioButtonGroup and RangeInput will need a similar approach.

Reply:
I'm not completely up to date on the details here, but I think CheckBoxGroup, RadioButtonGroup, and RangeInput should all have the right pad built into them such that they align with the other input elements.

Reply:
They do. However, the designs call for these inputs to occupy more space, most apparent in the active/hover states. The approach Brittany is proposing here is to transfer the correct padding from FormField and instead apply it on the input.
![Screen Shot 2020-04-24 at 9 30 03 AM](https://user-images.githubusercontent.com/1756948/80229834-4005af80-860e-11ea-9e5e-f45f5c724415.png)

Reply:
Since the padding is coming from `formfield` It looked like the picture below where the hover color was not covering the whole component as in the designs.  I purposed that we took that padding out and added it back into the checkbox component itself, so that way the hover color was able to fill the box around it. 
We can also discuss other ways around this. This was just one way I had thought of. 

![Screen Shot 2020-04-22 at 3 38 24 PM](https://user-images.githubusercontent.com/42451602/80232055-7f81cb00-8611-11ea-8caa-ba1283f529b9.png)

Reply:
Totally agreed! The padding should be on CheckBoxGroup and friends, not FormField.

Reply:
yay 😃sounds good!